### DISCUSSION TITLE
Sun walk

### DISCUSSION THREAD

**Discussion by UweLoyal** [RomRider/apexcharts-card#800] [Q&A]
![image](https://github.com/user-attachments/assets/43d0b184-3dc2-42b1-ae42-851fbee4557d)

**Is it possible to set a Point (the sun) on the line?
It is an data point.**

Like this:
![image](https://github.com/user-attachments/assets/8a14efa4-0d01-4031-acb5-34475ee50f0a)

`type: custom:apexcharts-card
experimental:
  color_threshold: true
graph_span: 24h
update_interval: 2m
now:
  show: true
  label: Jetzt
  color: red
span:
  start: day
yaxis:
  - id: primary
    decimals: 2
    max: 70
    min: -70
header:
  show: true
  title: Sonnenverlauf
  show_states: false
  colorize_states: true
all_series_config:
  unit: °
  type: line
  color: black
  stroke_width: 0
  show:
    legend_value: false
    in_header: true
    header_color_threshold: true
series:
  - entity: sensor.Astro_Daten_Sonne_SunAltitude24h
    type: line
    stroke_width: 2
    name: Altitude dayline
    show:
      extremas: true
      in_header: before_now
    data_generator: |
      var data = [];
      var formattedJSON = JSON.parse(entity.state).data;
      console.log(formattedJSON);
      formattedJSON.forEach(obj => {
        data.push([obj.timestamp, (obj.value)])
      });
      return data;
  - entity: sensor.Astro_Daten_Sonne_NowSunAltitude
    name: Altitude now
`

**Comment by claymichaels** [comment]
Necropost for anyone else looking:

The Horizon Card in HACS works for me, and looks very similar to your examples.

https://github.com/rejuvenate/lovelace-horizon-card

<img width="984" height="550" alt="image" src="https://github.com/user-attachments/assets/891fba02-f6b9-486f-afde-9354944c626d" />

<img width="984" height="790" alt="image" src="https://github.com/user-attachments/assets/eeacc03f-0fa5-4280-a9f7-b13ba725597c" />
